usecases/MinimumWorkingExample/dodo.py

```diff
@@ -120,4 +123,29 @@ def task_export_knowledgeGraph_emodul():
                 'file_dep': [metadata_file_path, processed_data_file_path],
                 'targets': [knowledge_graph_file],
                 'clean': [clean_targets]
-            }
\ No newline at end of file
+            }
+
+# perform calibration
+def task_perform_calibration():
+    # create folder, if it is not there
+    Path(calibrated_data_directory).mkdir(parents=True, exist_ok=True)
+
+
+
+    # TODO implement "expensive" option on all files!!!
+    knowledge_graphs_file = Path(knowledge_graphs_directory, cheap_example_name + '.ttl')
+
+    # current outputs
+    owl_file = Path(calibrated_data_directory, 'calibrationWorkflow.owl')
+    displ_list = Path(calibrated_data_directory, 'displacement_list_' + cheap_example_name + '.dat')
```

Comment:
why is the force and displ list an output of the calibration? Where is that defined?

Reply:
ProbEye was just exporting it somehow. I didnt put that much emphasis on that. I will see.